sample/Kmd.Logic.Gateway.Automation.Sample/Publish/publish.yml

```diff
@@ -30,7 +30,7 @@ Apis:
       ProductNames: [nexus-1]
       PolicyXmlFile: ./apis/nexus/policies.xml
       BackendLocation: http://nexus
-      IsCurrent: true
+      IsCurrent: false
```

Comment:
I think , this will give an error since you cant make first version Iscurrent set to false. The very first version has to be set true.

Reply:
I have checked that. It does not make problems.